dependent_type_theory.md

```diff
@@ -355,11 +355,11 @@ You can also pass types as parameters:
 ```lean
 #check fun (α β γ : Type) (g : β → γ) (f : α → β) (x : α) => g (f x)
 ```
-The last expression, for example, denotes the function that takes
```

Comment:
Since there's only one expression in this example, using "last" makes one think that something other than the full expression is being referred to.